packages/core/package.json

```diff
@@ -66,6 +66,7 @@
     "file-url": "~3.0.0",
     "get-port": "~5.0.0",
     "glob": "~7.1.2",
+    "ignore": "^5.1.8",
```

Comment:
Was the `^` intentional here? Or did you mean `~`?

Reply:
Wow, not using it. Will remove 😅